### PR TITLE
Docs: Add mobile-env to community projects

### PR DESCRIPTION
**Pull request by stefanbschneider**
## Description

Add [mobile-env](https://github.com/stefanbschneider/mobile-env) as new community project to the docs.

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

I recently added a new example notebook for using `stable-baselines3` v2.0.0 with mobile-env, which I want to share.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (update in the documentation)

## Checklist

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [x] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [x] I have updated the documentation accordingly.
- [ ] I have opened an associated PR on the [SB3-Contrib repository](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib) (if necessary)
- [ ] I have opened an associated PR on the [RL-Zoo3 repository](https://github.com/DLR-RM/rl-baselines3-zoo) (if necessary)
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

**Comment by stefanbschneider** [comment]
> I have updated the changelog accordingly (**required**).

Does this also hold for this change, ie, only updating the docs? Happy to put my name into the changelog :)

File: docs/misc/projects.rst [hunk 1]
```diff
@@ -197,3 +197,18 @@ A simple library for pink noise exploration with deterministic (DDPG / TD3) and
 | Authors: Onno Eberhard, Jakob Hollenstein, Cristina Pinneri, Georg Martius
 | Github: https://github.com/martius-lab/pink-noise-rl
 | Paper: https://openreview.net/forum?id=hQ9V5QN27eS (Oral at ICLR 2023)
+
+
+mobile-env
+----------------------
```
Comment (araffin) [inline docs/misc/projects.rst:203]:
```suggestion
mobile-env
----------
```

**Comment by araffin** [comment]
>  Happy to put my name into the changelog :)

yes, please also update the changelog ;)

**Review by araffin — APPROVED** [review]
LGTM, thanks =)

**Pull request merged**